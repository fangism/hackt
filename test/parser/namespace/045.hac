// "045.hac"
// expect fail: prs forbidden outside global namespace

namespace foo {
prs {

}
}	// end namespace foo

