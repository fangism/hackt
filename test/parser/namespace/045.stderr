Error: top-level PRS is only supported in the global namespace.
	got: prs { ... } [5:1--7:1]
